### PR TITLE
Columnar: cleanup

### PR DESCRIPTION
**Pull request by jeff-davis**
More columnar cleanup.

### CODE REVIEW

File: src/include/columnar/columnar_compression.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*-------------------------------------------------------------------------
+ *
+ * columnar_compression.h
+ *
+ * Type and function declarations for compression methods.
+ *
+ * Copyright (c) 2016, Citus Data, Inc.
+ *
+ * $Id$
```
Comment (onurctirtir) [inline src/include/columnar/columnar_compression.h:9]: Looked into other files in codebase, I guess we should not have `$Id$` and copyright years anymore

```suggestion
 * Copyright (c) Citus Data, Inc.
```

File: src/include/columnar/columnar_metadata.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*-------------------------------------------------------------------------
+ *
+ * columnar_metadata.h
+ *
+ * Type and function declarations for Columnar metadata.
+ *
+ * Copyright (c) 2016, Citus Data, Inc.
+ *
+ * $Id$
```
Comment (onurctirtir) [inline src/include/columnar/columnar_metadata.h:9]: same

```suggestion
 * Copyright (c) Citus Data, Inc.
```

**Review by onurctirtir — APPROVED** [review]
left two minor comments

File: src/include/columnar/columnar_compression.h (hunk 1 above)
Comment (jeff-davis) [reply src/include/columnar/columnar_compression.h]: Done.

File: src/include/columnar/columnar_metadata.h (hunk 2 above)
Comment (jeff-davis) [reply src/include/columnar/columnar_metadata.h]: Done.

**Pull request merged**